### DISCUSSION TITLE
cdn.jsdelivr.net 坏掉了

### DISCUSSION THREAD

**Discussion by shuax** [xmake-io/xmake#2380] [General]
简单的办法是改成 fastly.jsdelivr.net

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/170394969-098d3d6c-42db-4514-8547-a272db419af3.png)

**Reply by waruqi** [reply]
2.6.6 版本已经删了 jsdelivr ，但是你老版本更新只能全量自己Release装下
